File: Artificial Intelligence/Labs/2. Swiprolog/Lab 10/10.3.pl
greatest:-
write('Enter the first number : '),
read(X),
write('Enter the second number : '),
read(Y),
write('Enter the third number : '),
read(Z),
maxi(X,Y,Z).
maxi(A,B,C):-
A=<B,B=<C,
write('The maximum number is : '),write(C).
maxi(A,B,C):-
A>=B,A>=C,
write('The maximum number is : '),write(A).
maxi(A,B,C):-
write('The maximum number is : '),write(B).

File: Artificial Intelligence/Labs/2. Swiprolog/Lab 10/10.3.cpp
greatest():-
write("Enter the first number : "),
read(X),
write("Enter the second number : "),
read(Y),
write("Enter the third number : "),
read(Z),
maxi(X,Y,Z).
maxi(X,Y,Z):-
Y>Z,X>Y,
write("The maximum number is : "),write(X).
maxi(X,Y,Z):-
Z>X,X>Y,
write("The maximum number is : "),write(Z).
maxi(X,Y,Z):-
write("The maximum number is : "),write(Y).